miner/block_builder.go

```diff
@@ -42,7 +42,7 @@ type AtxValidator interface {
 
 type Syncer interface {
 	FetchPoetProof(poetProofRef []byte) error
-	IsSynced() bool
+	WeaklySynced() bool
```

Comment:
Do we have a "StronglySynced" notion?

Reply:
Yes, "FullySynced" tough only on paper and not in code.
The code simply continues with the term IsSynced.